staging/src/k8s.io/sample-apiserver/hack/update-codegen.sh

```diff
@@ -67,3 +67,5 @@ ${informergen} \
   --listers-package k8s.io/sample-apiserver/pkg/client/listers \
   --output-package k8s.io/sample-apiserver/pkg/client/informers
   "$@"
+
+cp -R $GOPATH/src/k8s.io/sample-apiserver/pkg/client $GOPATH/src/k8s.io/kubernetes/staging/src/k8s.io/sample-apiserver/pkg/  
```

Comment:
It does currently correct. It generates at ```$GOPATH/src/k8s.io/sample-apiserver/pkg/client``` I will look through the script commands and see can I improve this.